vertical-pod-autoscaler/pkg/updater/priority/update_priority_calculator.go

```diff
@@ -84,7 +84,7 @@ func NewUpdatePriorityCalculator(vpa *vpa_types.VerticalPodAutoscaler,
 func (calc *UpdatePriorityCalculator) AddPod(pod *apiv1.Pod, now time.Time) {
 	processedRecommendation, _, err := calc.recommendationProcessor.Apply(calc.vpa.Status.Recommendation, calc.vpa.Spec.ResourcePolicy, calc.vpa.Status.Conditions, pod)
 	if err != nil {
-		klog.V(2).Infof("cannot process recommendation for pod %s: %v", klog.KObj(pod), err)
+		klog.V(2).InfoS("Cannot process recommendation for pod", "pod", klog.KObj(pod), "error", err)
```

Comment:
Seems like this should be a call to `.ErrorS` instead?

Reply:
Yeah I agree